scripts/export.py

```diff
@@ -20,6 +20,9 @@
 OUTLIER_TYPE_FIELD = "USER27"
 RUN_ID_FIELD = "USER28"
 
+IS_OUTLIER_INDICATOR = "N: Non arms-length"
+NON_OUTLIER_INDICATOR = "Y: Is arms-length"
+
 OUTLIER_TYPE_CODES = {
     "Not outlier": "0",
```

Comment:
Do we also want to use the long value for the outlier type, in order to make that field read-only? I notice that the iasWorld data model does not have an "analyst determination reason" field in addition to "model determination reason", as "model determination"/"analyst determination" does, so perhaps we want this field to be writeable, but I also wonder if that was an oversight in our iasWorld schema design.

Reply:
We do actually want the determination reason to be writable, as it's basically shared between the model / analyst determination fields.